.github/workflows/build.yml

```diff
@@ -29,15 +29,10 @@ jobs:
       - name: Install Node.js modules
         run: |
           npm install -g pnpm@6.2.3
-          pnpm install --unsafe-perm --frozen-lockfile
```

Comment:
You removed the lock file which pins the dependencies?  We need to ensure everything is pinned.

Reply:
Fixed in the latest commit